app/plugins/views.js

```diff
@@ -21,6 +21,7 @@ module.exports = {
             'node_modules/govuk-frontend/dist'
           ], {
             autoescape: true,
+            trimBlocks: true,
```

Comment:
What's this for? Digging around, for hapi/vision this changes the behaviour of text and new lines in some cases, is this wanted?

Reply:
This is actually [nunjucks configuration](https://mozilla.github.io/nunjucks/api.html#configure) it's partially successful at trimming newlines from the end of template blocks when rendering HTML. (It seems to pad them all with a couple of them by default). Which makes reading the debug output from testing library a bit easier.

Reply:
Yep just wondered why we needed it setting to true now, as it wasn;t before. As long as it doesn't mess with the (1?) actual page rendered by the app then now worries